pettingzoo/utils/random_demo.py

```diff
@@ -3,26 +3,20 @@
 import numpy as np
 
 
-def random_demo(env, render=True, cycles=100000000):
+def random_demo(env, render=True, episodes=1):
     '''
     Runs an env object with random actions.
     '''
 
-    env.reset()
-
-    if hasattr(env, 'display_wait'):
-        display_wait = env.display_wait
-    else:
-        display_wait = 0.0
-
     total_reward = 0
     done = False
+    run_episodes = 0
 
-    for cycle in range(cycles):
+    while True:
+        env.reset()
         for agent in env.agent_iter(len(env.agents)):
             if render:
                 env.render()
-                time.sleep(display_wait)
```

Comment:
idk how I feel about this, I think most environments can render faster than 60fps, which makes them hard to watch as a demo. If you add a `time.sleep(0.015)` It would enforce something close to 60 fps.